File: lab1-semester1/2.cpp
/*2	Declaring Variable and Printing its Value
(Hint: one variable of each type like: int ,float ,double ,char).*/

#include<iostream>
using namespace std;

int main(){
    int num1=1; //int takes 4 bytes for storage
    float num2=1.1; //float takes 4 bytes for storage
    double num3=1.2; //double takes 8 bytes for storage
    char grade ='A'; //char takes 1 bytes for storage
    
    cout<<" int : "<<num1<<endl;
    cout<<" float : "<<num2<<endl;
    cout<<" double : "<<num3<<endl;
    cout<<" char : "<<grade<<endl;
    
    system("pause");
    return 0;
}

File: lab1-semester1/2.go
/*2	Declaring Variable and Printing its Value
(Hint: one variable of each type like: int ,float64 , complex128 , byte, rune, string).*/

package main

import (
	"fmt"
)

func main() {

var num1 int=1
var num2 float64=11.11//float64 for float numbers
var num3 complex128=0.077+0.1i//complec128 is for complex numbers
var t byte='A'//byte ali
var f rune='£'// rune used to show UNICODE characters
var x string="This is hard"

	fmt.Println(" int : ", num1)
	fmt.Println(" float64 : ", num2)
	fmt.Println(" complex128 : ", num3)
	fmt.Printf(" byte :  %c", t)//this will print character A
	fmt.Printf("\n rune :  %c", f)//this will print pound sign
	fmt.Printf("\n string : %s", x)
}
